test/models/default-filters-test.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert')
+const ElectronConfig = require('electron-config')
+const storage = new ElectronConfig({ name: 'config-test' })
+const DefaultFilters = require('../../src/models/DefaultFilters')
+const login = 'RupertScrivener'
```

Comment:
O shit, this one's up for grabs, boys!

<img width="644" alt="screen shot 2016-09-11 at 4 34 45 pm" src="https://cloud.githubusercontent.com/assets/82317/18420669/b09dad7a-783d-11e6-971c-58988764a172.png">